jdaviz/core/template_mixin.py

```diff
@@ -3774,7 +3776,7 @@ def _on_glue_value_changed(self, value):
             value = value.name
         elif self._glue_name in GLUE_STATES_WITH_HELPERS:
             value = str(value)
-        elif isinstance(self.value, (int, float)) and self._glue_name != 'percentile':
+        elif self._glue_name != 'percentile' and isinstance(self.value, (int, float)):
```

Comment:
technically not related to this effort, but a small cleanup that will avoid the isinstance for all cases that aren't percentile.